server/database/models/subscription.ts

```diff
@@ -0,0 +1,21 @@
+import { DataTypes } from 'sequelize';
+import sequelize from '../config/connection';
+
+const Subscription = sequelize.define('Subscription', {
+  id: {
+    type: DataTypes.INTEGER,
+    autoIncrement: true,
+    primaryKey: true,
+  },
+  type: {
+    type: DataTypes.STRING,
+    allowNull: false,
```

Comment:
```suggestion
    allowNull: false,
     values: ['one', 'six'],
```
I would prefer to specify values either to be six for six months subscription or one